src/AWS.Deploy.Orchestration/DeploymentBundleHandler.cs

```diff
@@ -103,6 +104,13 @@ public async Task<string> CreateDotnetPublishZip(Recommendation recommendation)
             _interactiveService.LogInfoMessage(string.Empty);
             _interactiveService.LogInfoMessage("Creating Dotnet Publish Zip file...");
 
+            // Since Beanstalk doesn't currently have .NET 7 preinstalled we need to make sure we are doing a self contained publish when creating the deployment bundle.
+            if (recommendation.Recipe.TargetService == RecipeIdentifier.TARGET_SERVICE_ELASTIC_BEANSTALK && recommendation.ProjectDefinition.TargetFramework == "net7.0")
+            {
+                _interactiveService.LogInfoMessage("Using self contained publish since AWS Elastic Beanstalk does not currently have .NET 7 preinstalled");
```

Comment:
is there any way for us to keep track of when NET7 becomes available on beanstalk so we can remove this message? (at the very least create a backlog item for us to revisit this) and is there a public resource/link for customers to have an update on the timeline and .NET versions available on beanstalk?